src/Module/Notifications/Notify.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Friendica\Module\Notifications;
+
+use Friendica\BaseModule;
+use Friendica\Core\NotificationsManager;
+use Friendica\Core\System;
+use Friendica\Network\HTTPException;
+
+/**
+ * Interacting with the /notify command
+ */
+class Notify extends BaseModule
+{
+	/**
+	 * @throws HTTPException\InternalServerErrorException
+	 * @throws HTTPException\UnauthorizedException
+	 */
+	public static function rawContent()
+	{
+		if (!local_user()) {
+			throw new HTTPException\UnauthorizedException(L10n::t('Permission denied.'));
+		}
+
+		$a = self::getApp();
+
+		// @TODO: Replace with parameter from router
+		if ($a->argc > 2 && $a->argv[1] === 'view' && intval($a->argv[2])) {
```

Comment:
If this block doesn't return machine-readable output, then it should either be moved to `post()` or `content()` depending on the target request method.

Reply:
This is, what the `content` part is doing.
- `rawContent` get checked
- Fallback is `content`
- Redirect to `notification/system`

Reply:
What is the fallback to `content`? Why does it need to be done in `rawContent` since there's no possible machine-readable output in this block?

Reply:
Ah!! I got it now :D

Reply:
I'm not sure about that, why using the `init()` method since there's no possible overlap between `rawContent` and this block that really should be in `content()`?

Plus the error throw would end up in an unstyled error page, in `init()` it's better to just `return;` in case of exception, but in this case we don't even need an `init()`.

Reply:
Hm .. I thought because it isn't a "normal" content since we don't return anything .. for me, content is for real "output", not for redirection logic. But overlapping with `rawContent` is indeed a problem with it..